.github/workflows/new-issues.yml

```diff
@@ -9,12 +9,14 @@ permissions:
 jobs:
   automate-issues-labels:
     permissions:
-      issues: write  # for andymckay/labeler to label issues
-      pull-requests: write  # for andymckay/labeler to label PRs
+      issues: write
     runs-on: ubuntu-latest
     if: github.repository == 'llvm/llvm-project'
     steps:
-      - uses: andymckay/labeler@1.0.4
+      - uses: github/issue-labeler@v3.2
         with:
-          add-labels: 'new issue'
-          ignore-if-labeled: true
```

Comment:
Do we understand how this labeler behaves in presence of manually added labels?
We definitely don't want to fight with our tools again.

Reply:
The workflow only triggers when PRs are created, not when edited.
But to be sure, I've set sync-labels to false explicitly https://github.com/github/issue-labeler#syncing-labels